src/Raygun.NetCore.Blazor/Models/BrowserSpecs.cs

```diff
@@ -14,12 +14,10 @@ internal record BrowserSpecs
     {
         #region Private Members
 
-        private string calculatedBrowserVersion;
-        private string calculatedBrowserName;
-        private string calculatedOSVersion;
-        private string deviceModel;
-        private string deviceName;
-        private string browserManufacturer;
+        private string? _calculatedBrowserVersion;
+        private string? _calculatedBrowserName;
+        private string? _calculatedOSVersion;
+        // private string? _browserManufacturer;
```

Comment:
Should delete this unless we intend to wire it up

Reply:
At one point I had intended to wire those up from the UserAgentHints, but then I discovered that the versioning at that level is a dictionary of multiple sets of values, including an extra key designed to throw off people doing things like this. 

So I believe I decided to return the whole array (minus the BS extra key) instead, which means those can be deleted.